src/AI/agents/scripts/sync_prompts.py

```
@@ -0,0 +1,116 @@
+from agents.prompts.loader import PROMPTS_DIR, load_prompt
+from benchmarks.lf_api import LangfuseApi
+
+
+def _local_prompt_names() -> list[str]:
+    return sorted(path.stem for path in PROMPTS_DIR.glob("*.md"))
+
+
+def _remote(api: LangfuseApi, name: str) -> dict | None:
+    try:
+        return api._get(f"/api/public/v2/prompts/{name}")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)